common/changes/@uifabric/example-app-base/joem-textfield-add-setSelectRange_2017-05-18-21-16.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
It was the auto-output from rush. Is it cool to merge? I assume they just won't bump any of those packages anyway.